content/concepts/basics/03-content-relations.md

```diff
@@ -4,17 +4,68 @@ metaTitle: "sensenet - Content Relations"
 metaDescription: "sensenet Content Relations"
 ---
 
-# Content relations
+In sensenet the relation between contents is as much important as storing them. Because of the hierarchical tree structure of the content repository, it's important to mention parent and child relations between contents allowing users to take advantage of inheritance. Beside that, sensenet also gives the possibility to create other type of relations between contents. Such relation could be in a form of reference field (link to another content), binary field (for storing binary data on a content) or attachment (by combining reference and binary field).
+
 # Content repository tree
-# Parent
-# Child
-# Links
-https://wiki.sensenet.com/Reference_Field.html (nyilván innen értelemszerűen csak maga a tárolási/működési elv, ami kell)
-# Multiple links
-https://wiki.sensenet.com/Reference_Field.html (nyilván innen értelemszerűen csak maga a tárolási/működési elv, ami kell)
+
+sensenet content repository is basically a tree structure of the various stored content (e.g. folders, documents, user, tasks, etc.).
+This structure lets you organize, group your content easily by creating separate subtrees. It's not only important because of the possibility to configure content accessibility for different types of users on different parts of the tree, but also because of the inheritance of several things like content types' underlying business logic, metadata field configurations and settings.
+
+# Parent-child relation
+Content can inherit multiple things from its ancestors, from fields to permission the possibilities are basically infinite.
+
+## Permission inheritance
+Set a permission on a container item on a higher level (e.g. a document library or a workspace) it will be inherited by its children. This way you do not have to set permissions on every subfolder or document, because all content inherits permissions from their parent.
+
+## Settings inheritance
+In sensenet, settings are stored as content in the content repository so you can take all the advantages of the tree structure in case of settings as well.
```

Comment:
```suggestion
In sensenet, settings are stored as content in the content repository so you can take all the advantages of the tree structure in case of settings as well. Let's have watermark settings as an example. In sensenet it is possible to set a watermark globally for each and every preview image. In addition to that, you can define different watermarks for different workspaces (which will then overwrite global settings) or even set watermark to a specific doclib.

> ([learn more about settings](/concepts/basics/07-settings))
```